aws/resource_aws_glue_catalog_table_test.go

```diff
@@ -12,6 +12,41 @@ import (
 	"github.com/hashicorp/terraform/terraform"
 )
 
+func TestAccAWSGlueCatalogTable_recreates(t *testing.T) {
+	resourceName := "aws_glue_catalog_table.test"
+	rInt := acctest.RandInt()
+
+	resource.Test(t, resource.TestCase{
```

Comment:
We prefer to use the [`resource.ParallelTest()`](https://godoc.org/github.com/hashicorp/terraform/helper/resource#ParallelTest) function in this provider. 👍 

```suggestion
	resource.ParallelTest(t, resource.TestCase{
```